### PR TITLE
Re-remove TRUE and true IF

### PR DESCRIPTION
**Pull request by eileenmcnaughton**
Overview
----------------------------------------
Re-remove TRUE and true IF

Before
----------------------------------------
There was an IF that by definition was always true

After
----------------------------------------
It's gone

Technical Details
----------------------------------------
@demeritcowboy this is the do-over - I got the else & elseif this time

Comments
----------------------------------------

### CODE REVIEW

File: templates/CRM/Contribute/Form/Contribution/MainMembershipBlock.tpl [hunk 1]
```diff
@@ -8,108 +8,71 @@
...
+      } else if ( autoRenewOption == 2 ) {
+        autoRenewCheckbox.prop('checked',  true );
+        autoRenewCheckbox.attr( 'readonly', true );
+        autoRenew.hide();
+        forceRenew.show();
+      }
+    });
+  </script>
+{/literal}
 </div>
```
Comment (eileenmcnaughton) [inline templates/CRM/Contribute/Form/Contribution/MainMembershipBlock.tpl:112]: now I'm confused - I didn't intentionally remove a dif or fieldset & they still seem to match up but it shows that I have removed a couple of closers?

Comment (eileenmcnaughton) [reply templates/CRM/Contribute/Form/Contribution/MainMembershipBlock.tpl:112]: hmm local diff has them inside the if

image

Comment (seamuslee001) [reply templates/CRM/Contribute/Form/Contribution/MainMembershipBlock.tpl:112]: It is possible github being github and just picking the wrong </div> to say is removed

**Comment by demeritcowboy** [comment]
Maybe we can't handle the truth.

But it looks ok to me, the counts of pairs match, and the contribution page loads and doesn't look wonky, and let's me buy a membership. So let's do it.

**Comment by eileenmcnaughton** [comment]
TRUE dat

**Pull request merged**